app/src/test/java/nerd/tuxmobil/fahrplan/congress/models/SessionTest.kt

```diff
@@ -301,7 +301,9 @@ class SessionTest {
             changedIsNew = false
             changedIsCanceled = true
         }
-        assertThat(session.apply { cancel() }).isEqualTo(canceledSession)
+        val canceledSession = session.cancel()
+        assertThat(canceledSession).isEqualTo(comparableCanceledSession)
+        assertThat(session).isNotSameInstanceAs(canceledSession)
```

Comment:
This should be the other way round.

```suggestion
        assertThat(canceledSession).isNotSameInstanceAs(session)
```

Reply:
I can certainly swap these. Is there a particular reason besides readability/consistency?

Reply:
`session` is an input. But in a test you want to assert that the output meets certain criteria.